spec/javascripts/saveSpec.js

```diff
@@ -255,4 +254,21 @@ describe('Saving a resource instance', function() {
     });
 
   });
+
+  describe('payload for save requests', function() {
+    var resource;
+    beforeEach(function() {
+      resource = Model.create({ name: 'foo', id: 12, subject: 'hello world!' });
+    });
+
+    it('saves requested fields when specified', function() {
+      resource.save({fields: ['name', 'id']});
```

Comment:
not to mention the fact that the observers will pick up changes from fetching an updated copy of the resource, so we'd need extra logic to ignore those properties in the next save as well...